.github/workflows/phpunit-test.yml

```
@@ -0,0 +1,90 @@
+name: PHPUnit Test Runner
+
+on:
+  workflow_call:
+    inputs:
+      php-version:
+        required: false
+        type: string
+        default: '8.1'
```

```suggestion
        default: '8.2'
```

The _most used_ PHP version in WordPress websites is PHP 8.2.

Moreover, by the end of November 2025, PHP 8.1 will reach its _End of Life_.

Ref:
* https://wordpress.org/about/stats/
* https://www.php.net/supported-versions.php